Makefile

```diff
@@ -146,19 +148,22 @@ build/$(CONVERTER_JAR): $(CONVERTER_SOURCES) $(RESOURCES)
 %.class: %.java
 	$(JAVAC) -source $(JAVA_TARGET) -target $(JAVA_TARGET) -Xlint:-options -g:none $^
 
-test: all
-	test/smoke-test.sh
-	test/thread-smoke-test.sh
-	test/alloc-smoke-test.sh
-	test/load-library-test.sh
-	test/fdtransfer-smoke-test.sh
-	echo "All tests passed"
+test: all build/$(TEST_JAR)
+	echo "here"
+	echo $(LIB_PROFILER)
+	echo $(SOEXT)
```

Comment:
Please cleanup debug printing. You may leave an informational message here though, e.g.
`Running tests against $(LIB_PROFILER)`